libs/sdk-ui-geo/src/core/geoChart/helpers/geoChart/data.ts

```diff
@@ -157,31 +218,35 @@ function getBucketItemNameAndDataIndex(dv: DataViewFacade): IGeoData {
     }, {});
 
     // init data
-    const result: IGeoData = {};
+    const result: IGeoDataBuckets = {};
```

Comment:
All of these changes are just reformat after `BucketNames.LATITUDE and BucketNames.LONGITUDE` was added into the `forEach` array.